.claude/scripts/check-review-md.sh

```
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+# Check that paths and packages cited in .claude/REVIEW.md still exist.
+# Exits 1 if any cited path/package is missing, 0 otherwise.
+# Designed to be runnable locally and from CI.
+
+set -uo pipefail
+
+REVIEW="${1:-.claude/REVIEW.md}"
+REPO_ROOT="$(git rev-parse --show-toplevel 2>/dev/null || pwd)"
+cd "$REPO_ROOT"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)